### ISSUE TITLE
No divider between math and science

### ISSUE THREAD

**Issue by aronasorman** [learningequality/ka-lite#4777]
branch: develop
OS: Mac
Browser: Firefox 43

![screen shot 2016-01-05 at 08 54 50](https://cloud.githubusercontent.com/assets/191955/12118060/0ecb3d32-b38a-11e5-961c-64738b575150.png)

**Comment by dylanjbarth** [comment]
OS: mac

not an issue on chrome 47
![screen shot 2016-01-31 at 8 39 39 am](https://cloud.githubusercontent.com/assets/1319950/12701120/53808ba0-c7f6-11e5-8ccf-566929d1920f.png)

firefox 28 (haven't used ff in awhile!)
![screen shot 2016-01-31 at 8 47 19 am](https://cloud.githubusercontent.com/assets/1319950/12701145/57f2b9fa-c7f7-11e5-93d0-3eef5e413b0b.png)

firefox 40
![screen shot 2016-01-31 at 8 51 11 am](https://cloud.githubusercontent.com/assets/1319950/12701157/d2d46c4a-c7f7-11e5-858e-5ce6e08f6372.png)

and firefox 44
![screen shot 2016-01-31 at 8 57 05 am](https://cloud.githubusercontent.com/assets/1319950/12701178/a79d3132-c7f8-11e5-909c-8587ef971d63.png)

long story short, was unable to reproduce this @aronasorman

**Comment by aronasorman** [comment]
Thanks for checking! Closing.
